core/util.cjs

```diff
@@ -154,6 +154,10 @@ class Util {
             }
           }
         }
+
+        // TODO: convert printf-style displayValue.
```

Comment:
Is this TODO for this PR or later?

Reply:
It's an optional, p100 thing :P just something I noticed and wanted to document in the relevant place.